src/service.ts

```diff
@@ -3,7 +3,7 @@ import { existsSync } from 'fs';
 import { RequestHandler, createError, text, json, send } from 'micro';
 import { validateGithubSignature } from './githubSignature';
 import { mount } from './database';
-import { findMailboxId } from './findMailboxId';
+import { findMailbox } from './findMailboxId';
```

Comment:
```suggestion
import { findMailbox } from './findMailbox';
```

Reply:
I didn't change the file name because git couldn't show the diff anymore, but will now